src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/adapters/FolderAdapter.js

```diff
@@ -24,12 +24,13 @@ export default class FolderListAdapter extends React.Component<DatagridAdapterPr
         return (
             <FolderList onFolderClick={onItemClick}>
                 {data.map((item: Object) => (
-                    // TODO: We have to come up with a real solution here
+                    // TODO: Don't access properties like "title" directly.
+                    // Currently needed, will change with api update.
```

Comment:
I would remove that line, it confuses me more than it helps 😕 Or maybe change it to "use some kind of metadata instead"?